machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -51,7 +51,8 @@ The status of the Operator after installation depends on your cloud provider and
 |====
 [.small]
 --
-1. AWS clusters that are upgraded to version 4.12 from an earlier version require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+1. AWS clusters that are upgraded from version 4.11 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
+2. Azure clusters that are upgraded from version 4.12 or earlier require xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[CR activation].
```

Comment:
@sunzhaohua2 I updated this based on [your comment](https://github.com/openshift/openshift-docs/pull/55409#discussion_r1123975817) on the release notes. The table is a little easier to look at in the preview [here](https://55407--docspreview.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso-getting-started.html#cpmso-platform-matrix_cpmso-getting-started) :) PTAL and lmk if this is accurate.

Reply:
LGTM, thanks @jeana-redhat